baiji/s3.py

```diff
@@ -13,22 +13,13 @@
 a file from S3 into a local file.
 
 """
-import os
-import shutil
-import re
-from baiji.exceptions import InvalidSchemeException, S3Exception, KeyNotFound, BucketNotFound, KeyExists, get_transient_error_class
-from baiji.util.parallel import ParallelWorker
+from baiji.connection import S3Connection
```

Comment:
Is it worth pulling in the other libraries into this namespace? Technically it would be a breaking change, right? (Though I can see arguing that the helper classes weren't part of the public interface.)